app/errors/handlers.py

```diff
@@ -4,6 +4,15 @@
 
 
 # Error Handlers
+@bp.app_errorhandler(400)
+def bad_request(e):
+    error = dict(
+        code="bad-request",
+        message=str(e)
```

Comment:
Normally I advocate against passing along server error messages directly, but in the case of `400 - Bad Request`, I feel like the user should know why their request is bad. This seemed the simplest way to keep that usability, and I believe the security risk is rather low. But we need to make sure no one cargo cults this pattern for a more sensitive error